android-auto-app/src/main/res/values/strings.xml

```diff
@@ -2,4 +2,14 @@
     <string name="app_name">Dev Android Auto</string>
     <!-- TODO: Remove or change this placeholder text -->
     <string name="hello_blank_fragment">Hello blank fragment</string>
+
+    <string-array name="test_routes_junction_views" translatable="false">
+        <item>JCT</item>
+        <item>SAPA</item>
+        <item>CITYREAL</item>
+        <item>TOLLBRANCH</item>
+        <item>AFTERTOLL</item>
+        <item>EXPRESSWAY_ENTRANCE</item>
+        <item>EXPRESSWAY_EXIT</item>
```

Comment:
i can see these are copied over from the `MapboxJunctionActivity` so could be considered out of scope for this change.

but i think we should have human readable descriptions for these

Reply:
What human-readable description? These values are `TestRoutes` enum names.

Reply:
https://www.google.com/search?q=SAPA+mapbox+junction+view&ei=TZTJY-SCAcihiLMPi7CN-As&ved=0ahUKEwik3ZDZqdT8AhXIEGIAHQtYA78Q4dUDCBA&uact=5&oq=SAPA+mapbox+junction+view&gs_lcp=Cgxnd3Mtd2l6LXNlcnAQAzIFCCEQoAEyBQghEKABMgUIIRCgAToKCAAQRxDWBBCwAzoFCCEQqwJKBAhBGABKBAhGGABQjwZY1BlglxtoA3ABeAGAAXOIAbcKkgEEMTUuMZgBAKABAcgBCMABAQ&sclient=gws-wiz-serp

I don't know what SAPA means, do you? If you do I'm only suggesting that we add words